features/step_definitions/api-token.js

```diff
@@ -68,4 +68,51 @@ module.exports = function server() {
 
             Assert.notEqual(lastUsed, '');
         }));
+
+    this.Given(/^owns an existing API token named "([^"]*)"$/, tokenName =>
+        request({
+            uri: `${this.instance}/${this.namespace}/tokens`,
+            method: 'POST',
+            auth: {
+                bearer: this.jwt
+            },
+            body: {
+                name: tokenName
+            },
+            json: true
+        }).then((response) => {
+            Assert.oneOf(response.statusCode, [409, 201]);
+        }));
+
+    this.When(/^"calvin" lists all their tokens$/, () =>
```

Comment:
The only issue with that is that then the callback needs to have a parameter, otherwise cucumber will error.

There isn't really any logic that depends on the name though, so that leads to situations like [this](https://github.com/screwdriver-cd/screwdriver/blob/master/features/step_definitions/authorization.js#L35) where there's a `switch` statement that doesn't really do anything.

I felt like using a literal (as used in places like the [events](https://github.com/screwdriver-cd/screwdriver/blob/master/features/step_definitions/events.js#L55) tests) was probably cleaner.